pallets/phala/src/compute/computation.rs

```diff
@@ -40,25 +40,25 @@ pub mod pallet {
 	use fixed_sqrt::FixedSqrt;
 
 	const DEFAULT_EXPECTED_HEARTBEAT_COUNT: u32 = 20;
-	const MINING_PALLETID: PalletId = PalletId(*b"phala/pp");
+	const COMPUTING_PALLETID: PalletId = PalletId(*b"phala/pp");
 
 	#[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
-	pub enum MinerState {
+	pub enum WorkerState {
```

Comment:
Maybe call it "SessionState", short for "ComputingSessionSatate". Not quite sure. Just leave a comment for now